### ISSUE TITLE
Right-aligned label text is not flush when in fullscreen

### ISSUE THREAD

**Issue by sicienss** [godotengine/godot#30859]
**Godot version:**
3.1.1 stable (for both GLES3 and GLES2)

**OS/device:** Windows 8, Toshiba Satellite
**Graphics card:** Intel HD Graphics 4600
**Driver:** igdumdim64.dll,igd10iumd64,dll,igd10
**Driver version:** 10.18.10.3308 (9/16/2013)

**Issue description:**
When you create a `Label` with right-aligned text and custom `DynamicFont`, the text is perfectly flush when in windowed mode, but is not flush in fullscreen. Varying text length leads to different offsets from the right.

In the minimal example project, you'll notice that the labels themselves are perfectly flush (this is shown by the `ColorRect` children). But the text is far from flush, having a very ugly appearance.

Here's an image showing the issue in my project:

![image](https://user-images.githubusercontent.com/29932982/61989754-e291f300-afe8-11e9-8ee5-19d3ea059c0e.png)

**Steps to reproduce:**
1. Create two labels with any dynamic font of your choosing.
2. Set them to right-aligned.
3. Vary the length of the labels' text.
3. Observe a difference in alignment between windowed and fullscreen mode.

**Minimal Project:**
[TestProject.zip](https://github.com/godotengine/godot/files/3437824/TestProject.zip)

**Comment by homer666** [comment]
Strange, it seems to work fine for me in both master and 3.1.1, with both GLES3 and GLES2 renderers:

![image](https://user-images.githubusercontent.com/27100585/61991152-4ff85f00-b08f-11e9-8133-35764e37069b.png)

Godot versions: 3.2-dev 6f727c047 / 3.1.1-stable
OS: Windows 10 (build 18362)
Graphics card: NVIDIA Geforce 980 Ti
Graphics driver: NVIDIA display driver 430.64

If you're able to, please post your graphics card and driver version.

**Comment by sicienss** [comment]
@homer666, I edited my post to include graphics card info. I see the issue on both GLES2 and GLES3.

**Comment by CalebJohn** [comment]
I had a similar issue while testing my project on android. I was not able to reproduce this in fullscreen but could reproduce by changing the window size (see below).

![image](https://user-images.githubusercontent.com/2179547/72487407-219c4680-37cb-11ea-85c5-c75044b369db.png)
Godot version: 3.1.2 stable
OS: Ubuntu 18.04

I was fairly certain this was the same issue as #34799 which was fixed by #34838. So I tested with 3.2beta6
![b065ab4d054c724f149d6a4e0d33b709](https://user-images.githubusercontent.com/2179547/72487980-edc22080-37cc-11ea-9480-68eec82334d4.png)


Looks like this one is fixed!
